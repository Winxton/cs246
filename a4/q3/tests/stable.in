new 4
init
1 1
1 2
2 0
2 3
3 1
3 2
-1 -1
steps 10
print
