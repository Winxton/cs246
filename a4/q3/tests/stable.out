____
_XX_
X__X
_XX_
